packages/canary-react/src/stories/ic-data-table.stories.mdx

```diff
@@ -2081,7 +2081,7 @@ const DataTable = () => {
 
 ### Action Element
 
-An example with action elements passed in via the data
+Action Element and Action OnClick can be passed via the `data` prop to display in certain cells, this is beneficial when dealing with large sets of data.
```

Comment:
would be good to reference the names of the elements - something like:

The cell `data` can contain an `actionElement` which can be a string containing any html to be rendered. The `actionElement` will be displayed to the right of the cell data. An `actionOnClick` can be specified for the `actionElement`. This should be a function to be run when the `actionElement` is clicked.

am tempted to say remove the "this is beneficial ...." bit now